src/Microsoft.OData.Core/UriParser/Aggregation/AggregationMethod.cs

```diff
@@ -34,6 +34,12 @@ public enum AggregationMethod
         /// <summary>
         /// The aggregation method CountDistinct.
         /// </summary>
-        CountDistinct
+        CountDistinct,
+
+        /// <summary>
+        /// The aggregation method Count.
+        /// Used only internally to represent the virtual property $count.
+        /// </summary>
+        VirtualPropertyCount,

```

Comment:
If you take a look at the [aggregation doc](http://docs.oasis-open.org/odata/odata-data-aggregation-ext/v4.0/cs02/odata-data-aggregation-ext-v4.0-cs02.html#_Toc435016569), you will find that VirtualPropertyCount actually is not a method.

So I think it shouldn't be added here